packages/flame_tiled/lib/src/tiled_component.dart

```diff
@@ -73,7 +73,7 @@ class TiledComponent<T extends FlameGame> extends Component with HasGameRef<T> {
     if (tMap.orientation == MapOrientation.hexagonal) {
       if (tMap.staggerAxis == StaggerAxis.y) {
```

Comment:
Do you think this would be easier to read if it was switch case?

Reply:
Done.